src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -210,7 +211,11 @@ public boolean equals(Object other) {
         ModelManager otherModelManager = (ModelManager) other;
         return addressBook.equals(otherModelManager.addressBook)
                 && userPrefs.equals(otherModelManager.userPrefs)
-                && clients.equals(otherModelManager.clients);
+                && filteredClients.equals(otherModelManager.filteredClients)
+                && clients.equals(otherModelManager.clients)
+                && visibleRentalInformationList.equals(otherModelManager.visibleRentalInformationList)
+                && Objects.equals(lastViewedClient.get(), otherModelManager.lastViewedClient.get())
+                && commandHistoryStorage.equals(otherModelManager.commandHistoryStorage);
```

Comment:
Please add tests for the new equals method, should be quite straight forward.

Reply:
Implementing tests for the new `equals()` method is more complicated than expected because it would require refactoring of the way that `ModelManagerTest.java` sets up its tests.

I suggest I create a new issue for refactoring `ModelManagerTest.java` and address it in v1.6.